website/content/en/docs/User Guide/Cost Explained/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Breaking Down Cloud Operations Sandbox Cost"
+linkTitle: "Sandbox Cost Explained"
+weight: 2
+---
```

Comment:
For some reason this entire page seems to be rendered in a blue box

![image](https://user-images.githubusercontent.com/3914119/119874644-b2fcec00-beda-11eb-98fd-5d92fb95b965.png)